### PR TITLE
Add linters

### PR DESCRIPTION
**Pull request by saikatgpae**
This PR is created to add the ES6 version of Awesome books project.

### CODE REVIEW

File: module/deleteRow.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  const index = this.previousElementSibling.innerHTML;
+  const books = new Books();
+  const booksStored = JSON.parse(localStorage.getItem('books'));
+  books.list = booksStored;
+  books.removeItem(index);
+  localStorage.setItem('books', JSON.stringify(books.list));
+  /* eslint-disable */
+    location.reload();
+    /* eslint-enable */
+}
```
Comment (Kingstalux) [inline module/deleteRow.js]: - [x] Good job so far :+1: Kindly use `arrow` functions instead of regular functions :+1:

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @saikatgpae, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :clap: 
:ballot_box_with_check: App has the correct functionality :fire: 
:ballot_box_with_check: Linters are passing :ok_hand: 
:ballot_box_with_check: Code has been divided into modules :clap: 



## Required Changes ♻️
_Check the comments under the review._
- [x] Kindly personalise your README file with your information and information about your project and how a user can set up and run your project locally :100: 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: module/deleteRow.js (hunk 1 above)
Comment (saikatgpae) [reply module/deleteRow.js]: Changed the same.

**Review by mwanawabangona — APPROVED** [review]
Hi @saikatgpae  ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes !! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**